### ISSUE TITLE
incorrectly loading XSL, from file system while it's in classpath

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#59]
I can't use this JAR in eo-maven-plugin, here is what I see:

```
Caused by: java.io.FileNotFoundException: URL [jar:file:/Users/yb/.m2/repository/org/eolang/lints/0.0.10/lints-0.0.10.jar!/org/eolang/lints/aliases/alias-too-long.xsl] cannot be resolved to absolute file path because it does not reside in the file system: jar:file:/Users/yb/.m2/repository/org/eolang/lints/0.0.10/lints-0.0.10.jar!/org/eolang/lints/aliases/alias-too-long.xsl
      at io.github.secretx33.resourceresolver.ResourceUtils.getFile (ResourceUtils.java:215)
      at io.github.secretx33.resourceresolver.AbstractFileResolvingResource.getFile (AbstractFileResolvingResource.java:176)
      at io.github.secretx33.resourceresolver.UrlResource.getFile (UrlResource.java:276)
      at org.eolang.lints.XslLints.lambda$all$0 (XslLints.java:75)
```

We should add integration test (with invoker-maven-plugin) in order to reproduce this.

**Comment by h1alexbel** [comment]
@yegor256 reproduced with invoker integration test in #61. Full logs [here](https://github.com/objectionary/lints/actions/runs/12073961454/job/33671084222?pr=61). Apparently, this issues happens only on macOS system.

**Comment by h1alexbel** [comment]
@yegor256 actually, fixed in [9f00e90b](https://github.com/objectionary/lints/pull/61/commits/9f00e90b39bc5405aa9478089d4a4a82f208e518)

**Comment by yegor256** [comment]
@rultor release, tag is `0.0.11`
